pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -66,6 +67,7 @@ stdenv.mkDerivation rec {
     popd
 
     makeWrapper '${electron}/bin/electron' "$out/bin/teams-for-linux" \
+      --set LD_LIBRARY_PATH "${libpulseaudio}/lib" \
```

Maybe it would be better to `--prefix` instead of `--set`ting.
And also, use `lib.makeLibraryPath`.

So:
```suggestion
      --prefix LD_LIBRARY_PATH ${lib.makeLibraryPath [ libpulseaudio ]} \
```